_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -349,6 +356,35 @@ public static function register_endpoints() {
 		) );
 	}
 
+	public static function get_plans( $request ) {
+		$data      = get_transient( 'jetpack_plans' );
```

Comment:
Since the lines are now separated from each other, maybe we don't need the extra spaces before the `=` anymore? What do you think?